lib/data/external/AzureClient.js

```diff
@@ -98,6 +121,37 @@ class AzureClient {
             return callback(null, azureResp);
         });
     }
+
+    objectPutTagging(key, bucket, objectMD, log, callback) {
+        const azureKey = this._createAzureKey(bucket, key, this._bucketMatch);
+        const azureMD = this._getMetaHeaders(objectMD);
+        azureMD.tags = JSON.stringify(objectMD.tags);
+        this._client.setBlobMetadata(this._azureContainerName, azureKey,
+        azureMD, err => {
+            if (err) {
+                log.error('err from Azure GET data backend',
+                { error: err.message, stack: err.stack,
```

Comment:
I'm aware that this is not enforced everywhere, but the actual error object should be logged as the `error` attribute, the message as `errorMessage` and the stack as `errorStack`, otherwise indexing in ES will fail for a whole batch of messages.